### PR TITLE
cam6_3_099:  fix emissions in cam_dev

### LINKED ISSUE #759: drydep of gas phase species "skipped" in cam_dev
### What happened?

In incorporating PR cam6_3_083, I added a bug as a result of my new emissions reordering. The chemistry step in tphysac subtracts the drydep flux from the emissions array, to account for drydep. However, since in cam6_3-083, I moved the routine that applies the emissions to the state ahead of chemistry, the drydep flux is therefore not incorporated into the solution.

### What are the steps to reproduce the bug?

run cam_dev

### What CAM tag were you using?

cam6_3_092

### What machine were you running CAM on?

CGD machine (e.g. izumi)

### What compiler were you using?

Intel

### Path to a case directory, if applicable

_No response_

### Will you be addressing this bug yourself?

Yes

### Extra info

_No response_

### PR DESCRIPTION
**Pull request by adamrher**
Fixes #759

### CODE REVIEW

**Review by jtruesdal — COMMENTED** [review]
Looks good. One small comment correction.

File: src/chemistry/mozart/mo_gas_phase_chemdr.F90 [hunk 1]
```diff
@@ -1022,7 +1025,17 @@ subroutine gas_phase_chemdr(lchnk, ncol, imozart, q, &
     do m = 1,pcnst
        n = map2chm( m )
        if ( n > 0 ) then
-         cflx(:ncol,m)      = cflx(:ncol,m) - sflx(:ncol,n)
+         if (cam_physpkg_is("cam_dev")) then
+           ! apply to ptend array
```
[the remark this answers is not in the document]
Comment (brian-eaton) [reply src/chemistry/mozart/mo_gas_phase_chemdr.F90]: Fixed comment.

File: doc/ChangeLog [hunk 2]
```diff
@@ -1,5 +1,64 @@
...
 
+Tag name: cam6_3_099
+Originator(s): adamrher, eaton
+Date:
+One-line Summary: fix drydep emissions for cam_dev physics
+Github PR URL: https://github.com/ESCOMP/CAM/pull/763
+
+Purpose of changes (include the issue number and title text for each relevant GitHub issue):
+
+Fixes #759 (https://github.com/ESCOMP/CAM/issues/759)
```
Comment (cacraigucar) [inline doc/ChangeLog:11]: If you can copy the title from the issue here as well as the link, sometimes this give a little more information that users find useful

File: doc/ChangeLog [hunk 3]
```diff
@@ -1,5 +1,64 @@
...
+
+Describe any changes made to build system: none
+
+Describe any changes made to the namelist: none
+
+List any changes to the defaults for the boundary datasets: none
+
+Describe any substantial timing or memory changes: none
+
+Code reviewed by: jtruesdal, cacraigucar
```
Comment (cacraigucar) [inline doc/ChangeLog:21]: Don't forget to add the additional reviewers after they review

**Review by cacraigucar — APPROVED** [review]

**Review by jtruesdal — APPROVED** [review]

**Review by fvitt — APPROVED** [review]

**Review by nusbaume — APPROVED** [review]

File: doc/ChangeLog (hunk 2 above)
Comment (brian-eaton) [reply doc/ChangeLog:11]: Done.

File: doc/ChangeLog (hunk 3 above)
Comment (brian-eaton) [reply doc/ChangeLog:21]: Done.

**Pull request merged**